prov/tcp/src/xnet_attr.c

```diff
@@ -50,9 +50,10 @@
 
 #define XNET_TX_OP_FLAGS \
 	(FI_INJECT | FI_INJECT_COMPLETE | FI_TRANSMIT_COMPLETE | \
-	 FI_DELIVERY_COMPLETE | FI_COMMIT_COMPLETE | FI_COMPLETION)
+	 FI_DELIVERY_COMPLETE | FI_COMMIT_COMPLETE | FI_COMPLETION | \
+	 TCP_NO_CONNECT)
```

Comment:
Also unless I'm misunderstanding the code here, I don't think we want this to be in the default flags ? The default should be the current behavior, which is to attempt connection?